Backpack/Switch/Classes/BPKSwitch.h

```diff
@@ -26,6 +26,8 @@ NS_ASSUME_NONNULL_BEGIN
  */
 NS_SWIFT_NAME(Switch) IB_DESIGNABLE @interface BPKSwitch : UISwitch
 
+@property (nonatomic) UIColor *primaryColor500 UI_APPEARANCE_SELECTOR;
```

Comment:
I don't like this name. In React Native we call it `switchPrimaryColor` I think we should use the same name